rust/processor/src/config.rs

```diff
@@ -30,15 +29,15 @@ pub struct IndexerGrpcProcessorConfig {
     pub starting_version: Option<u64>,
     // Version to end indexing at
     pub ending_version: Option<u64>,
+    // Number of tables this processor writes to. This is used in the version tracker.
+    // Since DB table writes are parallel, it needs to know when all tables have been written to before it can move the latest version forward.
+    pub number_db_tables: u64,
```

Comment:
Having this requirement in config.yaml is not great. Thoughts on requiring a mapping for tablename -> # queries in the `ProcessorTrait`? On initialization, `VersionTracker` can count the total # queries per processor, excluding the ones in `skip_tables`.